src/components/shared/Heading.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+
+type HeadingProps = {
+  // headingClass?: string
+  headingText: string
+}
+type HeadingState = {
+}
+class Heading extends React.Component<HeadingProps, HeadingState> {
+  private static readonly textStyle = 'text-uppercase';
```

Comment:
weird approach for styling. Let's use separate SCSS file

Reply:
it will be fixed in the next mr